testsuite/billing.py

```diff
@@ -0,0 +1,59 @@
+"""APIs for Stripe and Braintree"""
+import braintree
+import stripe
+
+
+# pylint: disable=too-few-public-methods
+class Stripe:
+    """API for Stripe"""
+
+    def __init__(self, api_key):
+        # Due to fact that we can set up only one Stripe per 3scale and we use same api_key everytime this
+        # is not disruptive even if it looks like it is.
+        stripe.api_key = api_key
+
+    @staticmethod
+    def read_payment(customer_id):
+        """Read Stripe payment"""
+        return list(filter(lambda x: x['customer'] == customer_id, stripe.PaymentIntent.list()))[0]
```

Comment:
Isn't this banned as well?

Reply:
`next(x for x in stripe.PaymentIntent.list() if x['customer']  == customer_id)` would be much better